inter-rater_variability/02a_rootlets_to_spinal_levels.py

```diff
@@ -37,16 +40,17 @@ def get_parser():
 
     parser = argparse.ArgumentParser(
         description='The script does the following:'
-                    '\n\t- project the nerve rootlets on the spinal cord segmentation to obtain spinal levels'
+                    '\n\t- project the nerve rootlets on the spinal cord segmentation to obtain spinal levels '
+                    'OR projects the vertebral labels on the spinal cord segmentation to obtain vertebral levels.'
```

Comment:
```suggestion
                    'OR if vertebral labeling ("-type vertebral") is provided instead of the nerve rootlets, the script finds vertebral levels.'
```